cmd/machine-api-migration/main.go

```diff
@@ -80,6 +87,11 @@ func main() {
 		controllers.DefaultManagedNamespace,
 		"The namespace where CAPI components will run.",
 	)
+	mapiManagedNamespace := flag.String(
+		"mapi-namespace",
+		"openshift-machine-api",
```

Comment:
I think it'd be better to use a const for this

Reply:
Hmm - because there's no reasonable option other than`openshift-machine-api`?

Reply:
Ah sorry, I meant for storing the default value 😬 in case you need to reuse this string somewhere else where you don't have access to the flags. But we can refactor later if needed.

Reply:
Ohh - gotcha 🤣 That's sensible, will update :)